src/pages/gateway/api/reference/information.mdx

```diff
@@ -233,4 +234,93 @@ curl -X "POST" https://api-co-dev.placetopay.ws/gateway/interests \
 ```
         </CodeGroup>
     </Col>
-</Row>
\ No newline at end of file
+</Row>
+
+---
+
+## Solicitud de balance {{ id: 'balance-request', tag: 'POST', label: '/gateway/balance' }}
+
+<Row>
+    <Col>
+        Este servicio permite consultar la información de balance. Se debe consumir si el método de pago lo requiere (Cuando 'requireBalance' es 'true' en la respuesta obtenida del servicio de información).
```

Comment:
¿Por qué es diferente la traducción en inglés? Es cierto que es posible cambiar la forma de comunicar el mensaje según el idioma para dar mejor contexto, pero en esta ocasión veo diferencias en la información dada:

This service should be consumed if the payment method requires it (requireBalance flag set to true in the information service response).